mappings/net/minecraft/network/packet/play/PlayerInputPacket.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_2867 net/minecraft/network/packet/play/PlayerInputPacket
```

Comment:
shouldnt packets have a `C2S` or `S2C` in them?
or are these dual side packets?

Reply:
TBD. I discovered that there may be bidirectional packets so I'd need to figure out which ones actually were first. Left it out for now.

Reply:
alright then fair enough